src/TimeIntegration.cpp

```diff
@@ -170,7 +170,11 @@ void TimeIntegration::run() {
 #pragma acc update host(d_S_T[:bsize]) wait    // all in one update does not work!
 
             visual->visualise(m_solver, t_cur);
-            if (adaption->is_data_extraction_before_enabled()) adaption->extractData(adaption->get_before_name(), adaption->get_before_height(), t_cur);
+            if (adaption->is_data_extraction_before_enabled())
```

Comment:
if you change the one line to two lines, you should add { } at least

Reply:
done in ee47e67